scripts/docker/bokeh_docker_from_wheel.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# This installs Bokeh from a single wheel in the dist directory and runs some of the Python tests.
+
+set -eu
+
+echo "Start of $0"
+
+pip install dist/bokeh*.whl
+bash scripts/ci/install_node_modules.sh
+
+python -m bokeh info
+
+# Bokeh Python tests.
+bokeh sampledata && \
+pytest tests/unit -k "not firefox" && \
```

Comment:
This perhaps should now include `pytest tests/test_defaults`.

Reply:
Probably. I am still very much experimenting here, having to do it publicly as I can't test it locally.

Reply:
I have added this.